MdeModulePkg/Universal/BdsDxe/BdsEntry.c

```diff
@@ -832,6 +815,34 @@ BdsEntry (
   //
   InitializeLanguage (TRUE);
 
+  // MU_CHANGE [BEGIN]
+
+  //
+  // READ ONLY Variable (registered in mReadOnlyVariables) MUST BE be set before this point.
+  //
+
+  //
+  // Mark the read-only variables if the Variable Lock protocol exists
+  //
+  Status = gBS->LocateProtocol (&gEdkiiVariablePolicyProtocolGuid, NULL, (VOID **)&VariablePolicy);
+  DEBUG ((DEBUG_INFO, "[BdsDxe] Locate Variable Policy protocol - %r\n", Status));

```

Comment:
We can probably just print if we fail to find the protocol.

Reply:
Probably should even assert if you fail to find the protocol.

Reply:
Do we expect every edk2 based platform to produce the protocol at this point?

Reply:
I'd expect every Mu project to not every edk2 platform.